### PR TITLE
Mor shabi pan gke arm

### PR DESCRIPTION
**Pull request by MorShabiPAN**
Hi @rcaballeromx,

I couldn't make all the changes in your branch so I tried to copy the files and link changes to here and make my adjustments.

Can you please review, and share your thoughts?

Best,
Mor

This PR is following PR #476

### CODE REVIEW

File: compute/admin_guide/install/fragments/install_defender_twistcli_export_kubectl.adoc [hunk 1]
```diff
@@ -17,15 +17,15 @@ With YAML configuration files under version control, you can delete and reliably
 If you don't have `kubectl` access to your cluster, you can deploy Defender `DaemonSets` directly from the xref:../install/install_defender/install_cluster_container_defender.adoc[Console UI]. 
 
 This procedure shows you how to deploy Defender `DaemonSets` using the `twistcli` command-line utitily and declarative object management.
-You can also generate the installation commands using the Prisma Cloud Console UI under  *Manage > Defenders > Deploy > DaemonSet*.
-The installation scripts are only supported on Linux hosts.
-Use the `twistcli` command-line utility to generate the Defender `DaemonSet` YAML configuration files for macOS and Windows hosts. Deploy the cusjtom resources with `kubectl` following this procedure.
+You can also generate the installation commands using the Prisma Cloud Console UI under  *Manage > Defenders > Deploy > Defenders*.
+You can also use the `twistcli` command-line utility to generate the Defender `DaemonSet` YAML configuration files. Deploy the cusjtom resources with `kubectl` following this procedure.
```
Comment (sgordon46) [inline compute/admin_guide/install/fragments/install_defender_twistcli_export_kubectl.adoc]: cusjtom is mispelled.

**Comment by rcaballeromx** [comment]
Please review #476 instead. This content doesn't build properly.

**Review by rcaballeromx — CHANGES_REQUESTED** [review]
Please review #476 and ensure all the needed changes are included there. This PR doesn't build properly.

**Comment by rcaballeromx** [comment]
I added all the changes to the content present in this PR to #476. Please let me know if I missed any changes.

**Comment by Pubs-MV** [comment]
@rcaballeromx added all the changes to the content present in this PR to https://github.com/PaloAltoNetworks/prisma-cloud-docs/pull/476.

**Pull request closed without merging**